test/cmdlineTests/standard_empty_file_name/output.json

```diff
@@ -4,6 +4,10 @@
             "component": "general",
             "errorCode": "2904",
             "formattedMessage": "DeclarationError: Declaration \"A\" not found in \"\" (referenced as \".\").
+ --> :2:24:
+  |
+2 | pragma solidity >=0.0; import {A} from \".\";
+  |                        ^^^^^^^^^^^^^^^^^^^^
```

Comment:
Note that this affects also the errors reported by the assert in `CompilerStack::loadGeneratedIR()`.

Instead of this:
```
Invalid IR generated:
Error (6913): Call or assignment expected.

Error (8143): Expected keyword "data" or "object" or "}".
```
we'll now see this:
```
Invalid IR generated:
Error (6913): Call or assignment expected.
 --> :7:9:
  |
7 |         mstore(64, memoryguard(128))
  |         ^^^^^^

Error (8143): Expected keyword "data" or "object" or "}".
 --> :7:9:
  |
7 |         mstore(64, memoryguard(128))
  |         ^^^^^^
```

Another thing it affects are Yul test case variants that use the empty name for the source they parse and don't allow errors in expectations. The formatted errors are instead included in the exception they throw.